tests/selenium/conftest.py

```diff
@@ -560,14 +560,16 @@ def create_super_user(business_area: BusinessArea) -> User:
     call_command("loaddata", f"{settings.PROJECT_ROOT}/apps/geo/fixtures/data_small.json", verbosity=0)
     country = Country.objects.get(name="Afghanistan")
     business_area.countries.add(country)
-    user = UserFactory.create(
-        is_superuser=True,
-        is_staff=True,
-        username="superuser",
-        password="testtest2",
-        email="test@example.com",
-        partner=partner,
-    )
+    if not (user := User.objects.filter(pk="4196c2c5-c2dd-48d2-887f-3a9d39e78916").first()):
```

Comment:
🌵  what is this!??!

Reply:
@pavlo-mk

Reply:
in fixtures we have assigned  user with with this id `4196c2c5-c2dd-48d2-887f-3a9d39e78916` for rid grievances and a lot other models like created_by, imported_by. We need to have the user with this id

Reply:
can't we have a get or create?

Reply:
tried didn't work properly 
https://github.com/unicef/hope/pull/4477/commits/540bf6aed8ded226c5f2038cbab350f24ed3921a